sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -430,9 +430,12 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
           NextHttpPolicy nextPolicy,
           Context const& context) const override
       {
-        auto uuid = Uuid::CreateUuid().ToString();
+        if (!request.GetHeader(RequestIdHeader).HasValue())
```

Comment:
Add a changelog entry for these changes please, since it is an observable behavioral change. Maybe it's a bug fix?

Reply:
I don't think it is visible to the user: policies are internal, HTTP requests are being created by the Clients.
The SDK will continue to beave exactly as it was before.